jsref/Makefile

```diff
@@ -217,8 +216,9 @@ Translate_syntax.cmo: Translate_syntax.ml Translate_syntax.mli Translate_syntax.
 	ocamlc -c $(REPLACE_STDLIB) $(PPX_FLAG) $(OCAMLCFLAGS) $<
 
 clean_mljsref:
-	sh -c "rm -f $(ML_FRONTEND_SRC)/*.mli"
-	sh -c "rm -f {,$(ML_FRONTEND_SRC)/}*.{cmo,byte}"
+	rm -f $(ML_FRONTEND_SRC)/*.mli
```

Comment:
You seem to be missing one case, for the cmo and byte at top level (there are two curly groups in the second line).